### ISSUE TITLE
[BUG]: Vsync option gives variable frame times and framerate

### ISSUE THREAD

**Issue by IAmLiterallyInsane** [PCSX2/pcsx2#11725]
### Describe the Bug

I tried playing while using the "Sync to Host Refresh Rate" option and Vertical Sync on. When I check the OSD for frame times, they're too high for actual 60fps vsynced. They usually jump to 21-30ms instead of an expected ~16.7ms (perfect vsync). This basically makes games appear to be stuttering even though they run at 100%. I tried playing at 59.94 fps NTSC and 60 fps _without_ vsync enabled and the games run perfectly with no varying frame time or fps inconsistency.

### Reproduction Steps

Enable these options in the "Emulation" tab
![image](https://github.com/user-attachments/assets/fa1531c1-cbdc-4856-8aaa-2935e162d823)
Enable the OSD for fps and frame times
Start a game
Go fullscreen
Check OSD

### Expected Behavior

The fps and frame time "should" be consistent (as they are without vsync) but they both randomly jump, and are not the same. 
With VSync:
![image](https://github.com/user-attachments/assets/e9008cb5-1640-4c08-8653-4c7bac34e536)

Without VSync:
![image](https://github.com/user-attachments/assets/81de92a4-e5ee-4b56-a54c-efe423f78218)

(Game is Ratchet & Clank 1 NTSC-U)

### PCSX2 Revision

v2.0.2

### Operating System

Windows 10 (64bit)

### CPU

Ryzen 5 5600

### GPU

RTX 4060

### GS Settings

Basically just EE Cycle Rate at 180% (I think this is GS idk)

### Emulation Settings

Widescreen patches enabled 

### GS Window Screenshots

![image](https://github.com/user-attachments/assets/fb708074-0bf5-4806-803a-87b87980b5df)
![image](https://github.com/user-attachments/assets/3278cc40-6a9e-4011-94a0-1c23a8972822)
These don't really affect the game much since I have a good graphics card, the issue is mostly just with VSync.

**Comment by mrkenhoo** [comment]
I have the same problem. I am replying childhood games and Need for Speed Underground 2 is the worst game so far. ![Image](https://github.com/user-attachments/assets/2a748d8e-1117-4e97-a5e5-4b7d74725220)

**Comment by Mrlinkwii** [comment]
@mrkenhoo the issue your having isnt Vsync released , please use the discord or forum for support

**Comment by Mrlinkwii** [comment]
is this still an issue on latest nightly ?
